### PR TITLE
release: 2.67 changelogs

### PR DESCRIPTION
**Pull request by ernestl**
Merged master back into changelogs-2.67 branch which is based off tip of release/2.67.
This is required to introduce the commit history that includes the release tagged commit so that it is searchable on the master branch.

Merge changelogs back into master, please use a regular “merge” to merge it to retain commit history.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+* Make killing of snap apps best effort to avoid possibility of malicious failure loop
+* Alleviate impact of auto-refresh failure loop with progressive delay
+* Dropped timedatex in selinux-policy to avoid runtime issue
+* Fix missing syscalls in seccomp profile
+* Modify AppArmor template to allow using SNAP_REEXEC on arch systems
+* Modify AppArmor template to allow using vim.tiny (available in base snaps)
+* Modify AppArmor template to add read-access to debian_version
+* Modify AppArmor template to allow owner to read @{PROC}/@{pid}/sessionid
+* {common,personal,system}-files interface: prohibit trailing @ in filepaths
+* {desktop,shutdown,system-observe,upower-observe} interface: improve for Ubuntu Core Desktop
```
Comment (ZeyadYasser) [inline NEWS.md:39]:
```suggestion
* {desktop,shutdown,system-observe,upower-observe} interface: improvements for Ubuntu Core Desktop
```

File: NEWS.md
Comment (ZeyadYasser) [inline NEWS.md:1]: Do we want to mention the addition of backoff algorithm for failed auto-refreshes? https://github.com/canonical/snapd/pull/14578

**Review by ZeyadYasser — APPROVED** [review]
LGTM!

**Review by miguelpires — APPROVED** [review]
thanks

File: NEWS.md
Comment (ernestl) [reply NEWS.md:1]: The ship has sailed, this is just a sync to master. 
However, this is related: "Alleviate impact of auto-refresh failure loop with progressive delay"

**Pull request merged**